puppet/modules/kubernetes/manifests/apiserver.pp

```diff
@@ -1,6 +1,10 @@
 # class kubernetes::master
 class kubernetes::apiserver(
   $allow_privileged = true,
+  Boolean $audit_enabled = true,
+  String $audit_log_directory = '/etc/kubernetes/audit',
+  String $audit_log_path = '/etc/kubernetes/audit/kubernetes-audit.log',
```

Comment:
The log should be in `/var/log`

I suggest you create a subfolder with /var/log/kubernetes owned by kubernetes user and group